pkg/generic/fake/README.md

```diff
@@ -65,16 +65,9 @@ mock.EXPECT().List(gomock.Any()).Return(nil, nil)
 ```
 
 ## Fake Generation
-This package was generated with `mockgen -package fake -destination ./controller.go -source ../controller.go` and `mockgen -package fake -destination ./cache.go -source ../cache.go`
+This package was generated with `mockgen`, see [`generate.go`](./generate.go).
```

Comment:
Is it possible to include the command that was run to generate this file?

I believe this option is in the uber fork of this library

Reply:
The fork will automatically add a hint of the `mockgen` command used to generate it, but it's not really customizable.
Nonetheless, the only command needed must be `go generate` (just added it to the README). There is no need for users/developers to run `mockgen` command directly.